src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,48 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends BasePage {
+
+//    WebDriver driver;
+//    WebDriver wait;
+
+    //Locators
+    By emailField = By.cssSelector("[type='email']");
+    By passwordField = By.cssSelector("[type='password']");
+    By submitButton = By.cssSelector("button[type='submit']");
+
+    @FindBy(css = "[test]")
+    WebElement email;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation.